src/test/java/com/iota/iri/storage/rocksDB/LocalSnapshotsPersistenceProviderTest.java

```diff
@@ -0,0 +1,115 @@
+package com.iota.iri.storage.rocksDB;
+
+import com.iota.iri.conf.ConfigFactory;
+import com.iota.iri.conf.IotaConfig;
+import com.iota.iri.model.Hash;
+import com.iota.iri.model.HashFactory;
+import com.iota.iri.model.IntegerIndex;
+import com.iota.iri.model.LocalSnapshot;
+import com.iota.iri.model.persistables.SpentAddress;
+import com.iota.iri.service.snapshot.Snapshot;
+import com.iota.iri.service.snapshot.SnapshotProvider;
+import com.iota.iri.service.snapshot.impl.SnapshotMockUtils;
+import com.iota.iri.storage.LocalSnapshotsPersistenceProvider;
+import org.junit.*;
+import org.junit.rules.TemporaryFolder;
+
+public class LocalSnapshotsPersistenceProviderTest {
+    private final TemporaryFolder dbFolder = new TemporaryFolder();
+    private final TemporaryFolder logFolder = new TemporaryFolder();
+
+
+    private LocalSnapshotsPersistenceProvider provider;
+    private SnapshotProvider snapshotProvider;
+    private static final IntegerIndex LS_KEY = new IntegerIndex(1);
+    private static Hash milestoneHash = HashFactory.TRANSACTION.create("MILESTONE");
+
+
+
+    @Before
+    public void setup() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [You may have misspelled a JUnit framework method (setUp or tearDown)](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4082603)